src/Form/Type/Filter/DateTimeRangeType.php

```diff
@@ -46,7 +45,7 @@ public function configureOptions(OptionsResolver $resolver): void
             'operator_type' => DateRangeOperatorType::class,
             'field_type' => FormDateTimeRangeType::class,
             'field_options' => [
-                'field_options' => ['date_format' => DateTimeType::HTML5_FORMAT],
```

Comment:
I don't use Sonata anymore.

I kinda understand from the SF 7 failure that this option was useless ; but i'm not sure.

Can someone still using the lib confirm that it doesn't introduce a Bc break ? @phansys @dmaicher @jordisala1991 maybe ?

Reply:
I'll try to have a look over the weekend. I'm still using Sonata on one project but its not even close to being compatible with Symfony 7 :smile:

Reply:
You can help me by removing `'date_format' => DateTimeType::HTML5_FORMAT` in the DateTimeRangeType of your vendor even without SF7. I need to know if this option was useless ; since in SF7 it throw a logicException...

Reply:
As discussed via Slack there is only a tiny visual diff for me when removing that like. I think its fine to get rid of it.

Reply:
Let's merge then :)